common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -361,6 +391,84 @@ def test_can_reset_attempts(self):
         exam_attempts_section.remove_student_attempt()
         self.assertFalse(exam_attempts_section.is_student_attempt_visible)
 
+    @attr('a11y')
+    def test_track_selection_a11y(self):
+        """
+        Proctored: Track selection accessibility tests
+        """
+        self.track_selection_page.a11y_audit.config.set_rules({
```

Comment:
FWIW these work for me in their current state. The page object is established and available already. The `setUp` methods in other classes are when there's a separate class for the accessibility tests. Since we're including these checks within classes, we don't need to perform the set up again.

I was getting failures locally and on Jenkins. I excluded the tests that were failing that aren't part of this PR and fixed the ones that were relevant.